.github/workflows/release_pypi.yaml

```
@@ -21,6 +23,10 @@ jobs:
         with:
           python-version: '3.10'
 
+      - name: Bump Cargo version
+        run: |
+          python ../.github/scripts/cargo_version_bumper.py --target Cargo.toml "${{ github.ref_name }}"
```

Fails here: https://github.com/dottxt-ai/outlines-core/actions/runs/12237210355/job/34132430827?pr=111
```
Run python ../.github/scripts/cargo_version_bumper.py --target Cargo.toml "111/merge"
python: can't open file '/home/runner/work/outlines-core/outlines-core/../.github/scripts/cargo_version_bumper.py': [Errno 2] No such file or directory
Error: Process completed with exit code 2.
```
Remove the extra dot?
```suggestion
          python ./.github/scripts/cargo_version_bumper.py --target Cargo.toml "${{ github.ref_name }}"
```

---

🤦‍♂️